conf/kubernetes-resource-staging-server.yaml

```diff
@@ -1,6 +1,6 @@
 ---
-apiVersion: extensions/v1beta1
-kind: ReplicaSet
+apiVersion: apps/v1beta1
+kind: Deployment
```

Comment:
is Deployment compatible with k8s 1.5 (which as of now is "prior version" of k8s)?